packages/core/src/model.ts

```diff
@@ -716,6 +720,31 @@ export class Model implements Storage {
     return proto
   }
 
+  private getProperties (mixin: Ref<Class<Obj>>): PropertyDescriptorMap {
```

Comment:
I assume we already have similar descriptors after this.getPrototype() call, could we reuse them?